src/utils/Utils.php

```diff
@@ -675,4 +677,11 @@ function_exists('opcache_get_status') &&
 		//jit not available
 		return null;
 	}
+
+	public static function getRandomFloat(float $min = 0.0, float $max = 1.0) : float{
+		if($min > $max){
+			throw new \InvalidArgumentException("Minimum value cannot be greater than maximum value");
+		}
+		return $min + (mt_rand() / mt_getrandmax()) * ($max - $min);
```

Comment:
This is overengineered. These extra parameters are not needed.

Reply:
Removed. Just `Utils::getRandomFloat(): float`